lib/provider/bpmn/parts/IdProps.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(group, element, translate, options) {
   // Id
   group.entries.push(entryFactory.validationAwareTextField(translate, {
     id: options.id || 'id',
-    label: translate('Id'),
+    label: options.label || translate('Id'),
```

Comment:
What about translating here, as we already translate the description as part of the `IdProps`, too.

Reply:
The `option.label` attribute is defined in the `getIdOptions` of the properties provider as such :
```javascript
    return { id: 'participant-id', label: translate('Participant Id') };
```
So it already should be translated by the time it ends up in this function.
 (I don't know how to link to a file inside of the PR)

Reply:
https://github.com/bpmn-io/bpmn-js-properties-panel/pull/412/files#diff-9578e9e6d7be08c860d340b59799ea827673c391d9163b522c8199e7e9a5515fR135-R140

Reply:
Got it. So the assumption is: If something is passed, it is always translated already.

Reply:
However, we handle the `description` props differently.

These are translated, after the have been passed, cf. https://github.com/bpmn-io/bpmn-js-properties-panel/blob/e9f0346513f333d8804ab822a9a1a615835e531a/lib/provider/bpmn/parts/IdProps.js#L13.

Reply:
My proposal would be: We translate consistently, inside `*Props`, always.

Reply:
Alternative: Consistently translate _outside_ the `*Props`, always.